bin/deploy

```diff
@@ -359,6 +359,11 @@ main() {
     touch "${LSSTSW}/lfs/.git-lfs.deployed"
   )
 
+  # definition EUPS_PATH depending on the environment:
+  # including the last 7 characters of the environment name
+  export EUPS_PATH="${LSSTSW}/stack${LSST_CONDA_ENV_NAME: -7}"
```

Comment:
It may be better to remove the prefix rather than count characters in the suffix, but that part is OK.  If you do keep that, then you should probably make the same change to my code in `publish`.

I think it's better to set off the reference hash from the name of the stack by using `/stack-` rather than just `/stack`.

Reply:
Better using subfolders.
LSST_SPLENV_REF is available, so we can simplify.